content/posts/0000-webservers-rearchitecture.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Introducting new ways to bring your static files to the cloud
+date: "2022-05-16"
+slug: webservers-rearchitecture
+author: joshuatcasey
+---
+
+# Introducing the Paketo Web Servers Buildpack
+
+The Paketo team is pleased to introduce the [Web Servers buildpack](https://github.com/paketo-buildpacks/web-servers), available in the full builder as of [version 0.2.74](https://github.com/paketo-buildpacks/full-builder/releases/tag/v0.2.74).
+
+This buildpack allows you to serve static content using the popular NGINX or HTTPD web servers, with a variety of utilities for ease of use. Whether you have static files and a conf file that you want to package in an image, or you need to transform your dynamic content into static files and want the buildpack to generate the necessary conf file, the Web Servers buildpack has you covered.
+
+If the buildpack does not have the capability you need, please let us know on the [Paketo Slack](https://slack.paketo.io/) or by [filing an issue](https://github.com/paketo-buildpacks/web-servers/issues/new).
+
+While Paketo has had the [NGINX](https://github.com/paketo-buildpacks/nginx) and [HTTPD](https://github.com/paketo-buildpacks/httpd) buildpacks for some time, bringing them together in the new Web Servers buildpack enables the exciting capabilities listed below.
+
+## Using `npm` and `yarn` to generate static content
+
+You can now easily deploy your dynamic front-end application as static content, since the Web Servers buildpack allows you to execute arbitrary `npm` scripts by using the [`node-run-script`](https://github.com/paketo-buildpacks/node-run-script) buildpack.
+Just add appropriate scripts in `package.json` and use `BP_NODE_RUN_SCRIPTS` to tell the buildpack which scripts to run.
+
+This allows you to create your front-end application in the javascript framework of your choice and then serve it up using either `HTTPD` or `NGINX`.
+Since `npm` can be used in the absence of a javascript framework, you can also use this capability to download dependencies, transpile javascript, assemble content, or do anything necessary to translate your source material into static files.
```

Comment:
```suggestion
Since `npm` and `yarn` can be used in the absence of a javascript framework, you can also use this capability to download dependencies, transpile javascript, assemble content, or do anything necessary to translate your source material into static files.
```